### PR TITLE
QDB-3303 - Add wait for stabilization support.

### PR DESCRIPTION
**Pull request by vianneyPL**

### CODE REVIEW

**Review by mkurdej — APPROVED** [review]
LGTM.

**Comment by solatis** [comment]
Ok I may be completely blind, but where do you actually expose this function in java code?

File: src/main/c++/net/quasardb/qdb/jni/export/qdb_client.cpp [hunk 1]
```diff
@@ -258,6 +258,26 @@ Java_net_quasardb_qdb_jni_qdb_trim_1all(JNIEnv *jniEnv,
...
+                                                       jclass /*thisClass*/,
+                                                       jlong handle,
+                                                       jint timeout)
+{
+    qdb::jni::env env(jniEnv);
+    try
+    {
+        return jni::exception::throw_if_error(
+            (qdb_handle_t)handle,
+            qdb_wait_for_stabilization((qdb_handle_t)handle, timeout));
```
Comment (mkurdej) [inline src/main/c++/net/quasardb/qdb/jni/export/qdb_client.cpp:272]: @solatis, isn't that what you're looking for?

Comment (mkurdej) [reply src/main/c++/net/quasardb/qdb/jni/export/qdb_client.cpp:272]: Oh, never mind. I misread your comment.

Comment (vianneyPL) [reply src/main/c++/net/quasardb/qdb/jni/export/qdb_client.cpp:272]: I think he refers to this: https://github.com/bureau14/qdb-api-java/blob/2ce415b0498de3fc3db7900a1e33c194b307a872/src/main/java/net/quasardb/qdb/QdbCluster.java#L210
But it's in the java API instead, I'm not if it's the right approach.

Comment (solatis) [reply src/main/c++/net/quasardb/qdb/jni/export/qdb_client.cpp:272]: For clarity's sake, there is some functionality overlap between qdb-api-java and (the new) qdb-api-jni.  Suffice to say that new options and functionality should _always_ be added to the JNI API. The functionality in the java API is mostly there for backwards compatibility, e.g. our key/value API.

**Pull request merged**